### ISSUE TITLE
Log is flooded with INFO messages

### ISSUE THREAD

**Issue by alonpr** [snowflakedb/snowflake-kafka-connector#526]
The log is flooded with these INFO messages:

```
Creating Token with subject XXXXXXXXXXXXX.KAFKA_CONNECTOR (net.snowflake.ingest.connection.SecurityManager) [SecurityManager-1(15443)]
2023-01-18T12:38:32-05:00 2023-01-18 17:38:32,011 INFO Creating Token with issuer XXXXXXXXXXXXX.KAFKA_CONNECTOR.SHA256:JzwuAk5jo... (net.snowflake.ingest.connection.SecurityManager) [SecurityManager-1(15443)]
2023-01-18T12:38:32-05:00 2023-01-18 17:38:32,012 INFO Created new JWT (net.snowflake.ingest.connection.SecurityManager) [SecurityManager-1(15443)]
2023-01-18T12:38:32-05:00 2023-01-18 17:38:32,012 INFO Creating Token with subject XXXXXXXXXXXXX.KAFKA_CONNECTOR (net.snowflake.ingest.connection.SecurityManager) [SecurityManager-1(15444)]
2023-01-18T12:38:32-05:00 2023-01-18 17:38:32,012 INFO Creating Token with issuer
```

What are they about and is there a way to disable them? Maybe should be changed to TRACE?

**Comment by sfc-gh-tzhang** [comment]
@alonpr The particular one you share comes from the [Ingest SDK](https://github.com/snowflakedb/snowflake-ingest-java/blob/5cc84a208d0053df9fc28e79b6d0f392e1504e16/src/main/java/net/snowflake/ingest/connection/SecurityManager.java#L159) and they looks old so I assume these logging should be there if you switch back to an older version of connector, let us know if it's not true. On the other hand, we do have some logging [issue](https://github.com/snowflakedb/snowflake-kafka-connector/issues/517) and that's been fixed, it should go out with the next release.

**Comment by alonpr** [comment]
@sfc-gh-tzhang I'm using version 1.6.9

**Comment by sfc-gh-rcheng** [comment]
Hi @alonpr, though we haven't reduced the logs in the example you listed, this [PR](https://github.com/snowflakedb/snowflake-kafka-connector/pull/523) also reduces the number of INFO logs emitted by KC. It is available beginning version 1.8.3 (our latest is v1.9.1). 

I am investigating reducing the log level of the logs you listed

**Comment by sfc-gh-rcheng** [comment]
Merged a change to reduce the log level to DEBUG. @alonpr does this resolve your issue? If not please send me the full logs in which there are too many INFO messages
